internal/service/privatelinkendpoint/resource_privatelink_endpoint_test.go

```diff
@@ -25,12 +25,12 @@ func TestAccNetworkRSPrivateLinkEndpointAWS_basic(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:                 func() { acc.PreCheckBasic(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             testAccCheckMongoDBAtlasPrivateLinkEndpointDestroy,
+		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMongoDBAtlasPrivateLinkEndpointConfigBasic(orgID, projectName, providerName, region),
+				Config: configBasic(orgID, projectName, providerName, region),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckMongoDBAtlasPrivateLinkEndpointExists(resourceName),
+					checkExists(resourceName),
```

Comment:
seveeral acc tests have redundant checks like: 
					resource.TestCheckResourceAttrSet(resourceName, "provider_name"),
					resource.TestCheckResourceAttrSet(resourceName, "region"),

if later the value is checked with TestCheckResourceAttr